consensus/src/epoch_manager.rs

```diff
@@ -519,12 +519,18 @@ impl EpochManager {
                         monitor!("process_local_timeout", self.process_local_timeout(round).await)
                     }
                 }
-            ) {
-                counters::ERROR_COUNT.inc();
-                error!("{:?}", e);
-            }
-            if let RoundProcessor::Normal(p) = self.processor_mut() {
-                debug!("{}", p.round_state());
+            );
+            let round_state = if let RoundProcessor::Normal(p) = self.processor_mut() {
+                p.round_state().to_string()
+            } else {
+                "RoundState: None".into()
+            };
+            match result {
+                Ok(_) => trace!("{}", round_state),
+                Err(e) => {
+                    counters::ERROR_COUNT.inc();
+                    error!("{:?}, {}", e, round_state);
+                }
```

Comment:
Nice, hopefully this won't log so much.  Was this being hit so much just because a lot of messages were coming through?  Or was it a combination with this and the Round manager differences

Reply:
I haven't figured out why wrong dns would trigger the log spamming, it can only come from many more messages. probably need to dig further after this change.